test_runner/src/main/kotlin/ftl/run/platform/RunAndroidTests.kt

```diff
@@ -50,7 +40,10 @@ internal suspend fun AndroidArgs.runAndroidTests(): TestResult = coroutineScope
                 ignoredTestsShardChunks += context.ignoredTestCases
                 allTestShardChunks += context.shards
             }
-            context.reportPackageName()
+
+            if (args.appApk?.endsWith(".aab") != true) {
```

Comment:
I think we should only report if it ends with `.apk` as google may add additional extensions in the future

Reply:
Ok, what about `null` case, should we reportPackageName in this case? I guess we will get FileReference.remote == "" in this case

Reply:
What do you think about this approach?

If the app path doesn't end in `.apk`, we'll report an empty string. App path can't be null.

```diff
diff --git a/test_runner/src/main/kotlin/ftl/client/google/AppDetails.kt b/test_runner/src/main/kotlin/ftl/client/google/AppDetails.kt
index 0a806b70..118b74cd 100644
--- a/test_runner/src/main/kotlin/ftl/client/google/AppDetails.kt
+++ b/test_runner/src/main/kotlin/ftl/client/google/AppDetails.kt
@@ -3,10 +3,14 @@ package ftl.client.google
 import com.google.testing.model.FileReference
 import ftl.http.executeWithRetry

-fun getAndroidAppDetails(gcsAppPath: String): String =
-    GcTesting.get
+fun getAndroidAppDetails(gcsAppPath: String): String {
+    // getApkDetails errors when sent non-apk files such as aab
+    if (gcsAppPath.trim().lowercase().endsWith(".apk").not()) return ""
+
+    return GcTesting.get
         .ApplicationDetailService()
         .getApkDetails(FileReference().apply { gcsPath = gcsAppPath })
         .apply { requestHeaders.set("X-Server-Timeout", 1800) } // 30 min
         .executeWithRetry()
         ?.apkDetail?.apkManifest?.packageName?.toString().orEmpty()
+}
```

Reply:
Looks good to me